### PR TITLE
Slack - add notification types

### PR DESCRIPTION
**Pull request by liorblob**
## Status
- [ ] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

Added the notifications that are sent to the dedicated channel.

### CODE REVIEW

File: Packs/Slack/Integrations/Slack/README.md [hunk 1]
```diff
@@ -744,7 +744,16 @@ new incident [details]
...
 <h3>Notifications</h3>
-<p>The integration sends notifications according to the configured notification settings found in the user preferences page in Cortex XSOAR, and additionally to the pre determined channel dedicated for incident notifications (according to the integration configuration).</p>
+<p>The integration sends notifications according to the configured notification settings found in the user preferences page in Cortex XSOAR, and additionally to the pre determined channel dedicated for incident notifications (according to the integration configuration).
+If a dedicated channel for incident notifications is configured, the following notifications will be sent there:</p>
+<ul>
+<li>Incident opened</li>
+<li>Incident updated</li>
+<li>Investigation closed</li>
+<li>Investigation deleted</li>
+<li>Incident SLA</li>
```
Comment (Itay4) [inline Packs/Slack/Integrations/Slack/README.md:754]:
```suggestion
<li>Incident SLA changed</li>
```

File: Packs/Slack/README.md [hunk 2]
```diff
@@ -744,7 +744,16 @@ new incident [details]
 mirror incident-id
 </pre>
 <h3>Notifications</h3>
-<p>The integration sends notifications according to the configured notification settings found in the user preferences page in Cortex XSOAR, and additionally to the pre determined channel dedicated for incident notifications (according to the integration configuration).</p>
+<p>The integration sends notifications according to the configured notification settings found in the user preferences page in Cortex XSOAR, and additionally to the pre determined channel dedicated for incident notifications (according to the integration configuration).
```
Comment (Itay4) [inline Packs/Slack/README.md:747]: why do we have the readme here as well ?

Comment (liorblob) [reply Packs/Slack/README.md:747]: It's Slack + SlackAsk

Comment (Itay4) [reply Packs/Slack/README.md:747]: ok so lets keep this readme empty and have one integration for one for the script

**Comment by kirbles19** [comment]
@liorblob done.

**Pull request merged**